app/models/spree/order.rb

```diff
@@ -130,6 +130,14 @@ def states
       where("state != ?", state)
     }
 
+    # All the states an order can be in before completing the checkout
+    PRIOR_TO_COMPLETION_STATES = %w(cart address delivery payment).freeze
+    # All the states of a complete order but that shouldn't count towards the balance. Those that
+    # the customer won't enjoy.
```

Comment:
I'm not sure I know what you mean by the customer won't enjoy them? Not a big deal, obviously :)

Reply:
then it needs to be rephrased. It's a way to say that is a state that belongs after the order is placed but in which the customer won't have the products on their hand

Reply:
I'd appreciate it if you guys could help me find a perfect name for these two constants and the scope below. Naming is hard, you know.